instructions.md

```diff
@@ -0,0 +1,352 @@
+# Using the MASST+ binary
+
+The `masst_plus` binary integrates all tools needed to index and query mass spectrum database. It is designed to be fast and memory-efficient.
+
+The tool comes with two types of search: exact and error-tolerant (analog). Separate indices are needed for each search type. This tool is thus split into two subcommands: exact and analog, each of which has its own set of subcommands for creating, adding to, and querying indices.
+
+You can always consult `masst_plus --help` for detailed description of the available commands and flags. 
+
+## The search types
+
+**Exact search** mode matches query spectra against database spectra based on shared peaks only. The index subcommand ingests files into the disk-based database index. 
+
+**Analog search** mode (also known as error-tolerant) matches query spectra against database spectra based on sahred and shifted peaks. The index subcommand ingests files into the disk-based database index.
+
+## Creating Index
+
+To get started, create an spectrum index for the desired search type. The paramater available for each search type differ slightly, but both requires a directory for the index and some spectra file to add to the index.
+
+### Usage
+
+To create an index for **exact search**, run the following:
+
+```sh
+./masst_plus exact index [OPTIONS] --dir <DIR> --spectra-files <PATHS>...
+```
+or
+```sh
+./masst_plus exact index [OPTIONS] -d <DIR> -s <PATHS>...
+```
+
+Where `[OPTIONS]` could include the following. Note that `--peak-tol` and `--pepmass-tol` will be fixed for index. Flags under `Spectra` will be applied to preprocess the provided spectra files before they are added to the index, so they cannot be undone. 
```

Comment:
I would add that except for -d and -s users should feel free to leave all values at defaults if unsure